src/components/question-wrapper.tsx

```diff
@@ -235,7 +242,15 @@ export default class QuestionWrapper extends React.Component<IProps, IState> {
   private toggleExemplar = () => this.toggleTab("Exemplar");
 
   private toggleTab = (tabName: TabName) => {
+    const { sendCustomMessage } = this.props;
     const { activeTab } = this.state;
+    const messageType = tabName === "Correct" && activeTab !== "Correct"
+      ? "teacher-edition:showCorrectOverlay"
```

Comment:
Could create constants for these strings.

Reply:
Good call.  I added constants for the messages.  The tab names are already defined in the `TabName` type and used for the variables seen above so I think we're safe there:
`type TabName = "Correct" | "Distractors" | "TeacherTip" | "Exemplar";`